### ISSUE TITLE
Doesn't work in OCP (OpenShift Container Platform) Enterprise under 'restricted' SCC

### ISSUE THREAD

**Issue by garethahealy** [sonatype/docker-rhel-nexus#12]
Permissions error:

```
[root@rhel-cdk ~]# oc logs nexus3-3-aott8
id: cannot find name for user ID 1000040000
Unable to update instance pid: Unable to create directory /nexus-data/instances
/nexus-data/log/karaf.log (Permission denied)
Unable to update instance pid: Unable to create directory /nexus-data/instances
```

By default, pods in OCP run as the 'restricted' SCC, i.e.: locked down security.

Due to this, permissions on files need to be "fixed". Base images provided by RedHat for the products use the below script:

https://github.com/sclorg/s2i-base-container/blob/master/bin/fix-permissions

**Comment by garethahealy** [comment]
NOTE: If you are using the CDK, by default, this allows pods to run as "anyuid", which is less restrictive which might mean you haven't seen this problem previously.

**Comment by tchughesiv** [comment]
should be resolved w/
https://github.com/sonatype/docker-rhel-nexus/pull/29

**Comment by whyjustin** [comment]
Yes I believe this is fixed in #29 and #30. Thanks for this contribution @tchughesiv and thanks for pointing it out @garethahealy .
